README.md

```diff
@@ -60,7 +60,7 @@ Here is a server that closes the connection if you send it anything:
 $loop = React\EventLoop\Factory::create();
 $socket = new React\Socket\Server('127.0.0.1:8080', $loop);
 
-$socket->on('connection', function (ConnectionInterface $conn) {
+$socket->on('connection', function (\React\Socket\ConnectionInterface $conn) {
```

Comment:
For consistency with the other code, i'd remove the starting `\` from the namespace.

```suggestion
$socket->on('connection', function (React\Socket\ConnectionInterface $conn) {
```